   float64 s
   float64 t
  # speed (m/s)
   float64 v
  # acceleration (m/s^2)
   float64 a
  # jerk (m/s^3)
   float64 da